### PR TITLE
Upgrade to Jetty 11 maven plugin

### PR DESCRIPTION
**Pull request by snoopdave**
Also some attempts to reduce number of warnings about classes loaded from multiple locations, and some import cleanups. This change also makes it possible to run `mvn jetty:run` from the top-level directory and not only in the `app` module.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -25,10 +25,6 @@ limitations under the License.
     <version>6.1.0-SNAPSHOT</version>
     <packaging>pom</packaging>
 
-    <prerequisites>
```
Comment (snoopdave) [inline pom.xml:28]: This prerequisites tag is only to be used with Maven Plugin project POMs, not plain old modules like Roller's.

**Comment by mbien** [comment]
@snoopdave did you mean Jetty 10? Jetty 11 runs Jakarta EE 9 and most dependencies are not ready for that yet.

File: app/pom.xml [hunk 2]
```diff
@@ -55,7 +55,6 @@ limitations under the License.
         <maven-antrun.version>1.0b3</maven-antrun.version>
         <rome.version>1.13.1</rome.version>
         <slf4j.version>1.7.30</slf4j.version>
-        <spring.version>5.2.7.RELEASE</spring.version>
```
Comment (mbien) [inline app/pom.xml:58]: was this intended? spring and spring sec are now defined in two different poms. (asking before i rebase)

does this help to reduce selenium warnings or has it other reasons?

**Comment by snoopdave** [comment]
> @snoopdave did you mean Jetty 10? Jetty 11 runs Jakarta EE 9 and most dependencies are not ready for that yet.

Yes. Thanks! I edited the title to fix that.

File: app/pom.xml (hunk 2 above)
Comment (snoopdave) [reply app/pom.xml:58]: I was trying to use dependencyManagement in the parent POM to force all Spring deps to the same version, but I don't think it really helped with all those warnings you see when Jetty starts in app or it-selenium. 

I'm going to back this change out.

**Pull request merged**